src/sql/azdata.proposed.d.ts

```diff
@@ -892,6 +892,10 @@ declare module 'azdata' {
 		 * The selected ranges to be copied.
 		 */
 		selections: SelectionRange[];
+		/**
+		 * Whether to copy results from the UI process.
+		 */
+		copyFromUIProcess?: boolean;
```

Comment:
I can't see where this is being set.  Am I missing something?

Reply:
It's set here: https://github.com/microsoft/sqlops-dataprotocolclient/pull/101/files

Do the azdata.proposed.d.ts file in ADS have to match with what's in azdata.proposed.d.ts from the sqlops-dataprotocolclient? I added it, so that they match, but it's causing confusion.

Reply:
the typings definitions in that component only need to match if you're making updates that are consumed there, or non-backcompat breaking changes.  I'm surprised this would require changes there.  

Why isn't this parameter being set in the UI (query editor), it seems like that would provide the most flexibility for the client to determine how to perform the copy.  The UI client should have the most knowledge about how to do the copy, I think (e.g. regarding platform, runtime env, user options/preference, command entry point, etc.)?